### ISSUE TITLE
Simple Payments: Fix accessibility by adding appropriate WAI-ARIA tags

### ISSUE THREAD

**Issue by aldavigdis** [Automattic/wp-calypso#28801]
Simple Payments has a rather nonsemantic markup. This can be mitigated accessibility-wise by using the appropriate WAI-ARIA tags. This should be an easy fix.

See more at https://www.w3.org/WAI/PF/aria/states_and_properties#aria-labelledby

This issue is hereby split from #28781, as that ticket is already out of scope of this one and other things may depend on the current markup style.

**Comment by sirreal** [comment]
I think we should transfer this to https://github.com/Automattic/jetpack/ as that's where the HTML output produced is.
